apps/backend/src/datasources/UserDataSource.ts

```diff
@@ -20,6 +20,7 @@ export interface UserDataSource {
   // Read
   me(id: number): Promise<User | null>;
   getUser(id: number): Promise<User | null>;
+  getUsersByUserNumbers(id: readonly number[]): Promise<User[] | null>;

```

Comment:
For the returned type, it should be Promise<User[]>, as we will not receive null value